.github/workflows/build-and-publish.yml

```
@@ -17,19 +19,32 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Extract metadata (tags, labels) for Docker
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}
+          images: |
+            ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}
+            ghcr.io/${{ github.repository }}
           tags: |
-            type=ref,event=branch            
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
-            # set 'latest' tag for main branch
-            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
+            type=ref,event=branch  # Tag with branch name
+            type=raw,value=latest,enable=${{ github.ref == 'refs/heads/develop' }}  # Tag as 'latest' if it's the 'develop' branch
```

```suggestion
            type=raw,value=latest,enable=${{ github.ref == 'refs/heads/main' }}  # Tag as 'latest' if it's the 'main' branch
```
Looks like it was an intentional change from main to develop, but I think we want to keep latest pointing to main. Let me know if I missed something.

---

Or just remove `latest` altogether. I don't like the `latest` tag anyway.